iothub/service/src/Registry/DevicesClient.cs

```diff
@@ -567,7 +567,7 @@ public virtual async Task<IotHubJobResponse> ImportAsync(ImportJobProperties job
 
             try
             {
-                return await CreateJobAsync<ImportJobProperties>(jobParameters, cancellationToken).ConfigureAwait(false);
```

Comment:
I'm a little confused, now. I thought the point of the ImportJobProperties class was to be returned in this method. Are we actually using the importJobProperties class anymore?

Reply:
Fixed the change. 
The return type of importAsync / exportAsync should be ImportJobProperties / ExportJobProperties.
GetJobAsync() should return instance of IotHubJobResponse